lms/static/scripts/frontend_apps/components/ErrorDisplay.js

```diff
@@ -138,17 +119,21 @@ Technical details: ${details || 'N/A'}
   return (
     <Scrollbox classes="LMS-Scrollbox">
       <div className="hyp-u-vertical-spacing hyp-u-padding--top--4">
-        {children}
-        {description && (
+        {error.message && !description && (
           <p data-testid="error-message">
-            {description}
-            {error.message && (
-              <>
-                : <i>{toSentence(error.message)}</i>
-              </>
-            )}
+            <i>{toSentence(error.message)}</i>
           </p>
         )}
+        {error.message && description && (
+          <p data-testid="error-message">
+            {description}: <i>{toSentence(error.message)}</i>
+          </p>
+        )}
+        {!error.message && description && (
+          <p data-testid="error-message">{toSentence(description)}</p>
+        )}
```

Comment:
This results in the the same general formatting as before, but no longer requires the presence of `description` to render the `error.message`.